frontend/src/layout/navigation/TopBar/announcementLogic.ts

```diff
@@ -16,6 +17,9 @@ export enum AnnouncementType {
     AttentionRequired = 'AttentionRequired',
 }
 
+export const DefaultCloudAnnouncement =
+    "<p>We're experiencing technical difficulties, see more </p><a href='status.posthog.com'>status.posthog.com</a>"
```

Comment:
@timgl the idea is that I can just set the flag to be true (faster) and it will link to status.posthog.com e.g. while we're still investigating during the incident, is this text fine?